docs/source/smartdevice.rst

```diff
@@ -1,32 +1,32 @@
-.. py:module:: kasa
+.. py:currentmodule:: kasa
 
-Common API
-==========
+Base Device
+===========
 
 .. contents:: Contents
    :local:
 
-Device class
-************
+SmartDevice class
+*****************
 
-The basic functionalities of all supported devices are accessible using the common :class:`Device` base class.
+The basic functionalities of all supported devices are accessible using the common :class:`SmartDevice` base class.
```

Comment:
Shouldn't this remain as a description of the common device API?

Reply:
This file is now back to how it was pre the API changes.  I plan to copy the explanation text of this section into the `topics.md` in a subsequent PR.  If I add it in this PR it's too many changes and the rename `docs/source/design.rst → docs/source/topics.md` is lost.